src/relay/op/dyn/tensor/transform.cc

```diff
@@ -258,6 +258,7 @@ RELAY_REGISTER_OP("dyn.broadcast_to")
     .describe(R"code(Broadcast the first input to match the shape argument.
 )code" TVM_ADD_FILELINE)
     .set_num_inputs(2)
+    .set_attrs_type<InitOpAttrs>()
```

Comment:
Random change?

Reply:
no, needed since I include broadcast in my test relay text, and parsing fails without it. (Another glitch I fix-as-I-go.)